Sources/FoundationEssentials/ProcessInfo/ProcessInfo.swift

```diff
@@ -142,7 +142,11 @@ final class _ProcessInfo: Sendable {
         let time: UInt64 = ullTime
 #else
         var ts: timespec = timespec()
+#if os(Linux)
```

Comment:
This changes behaviour on Android I believe.

Reply:
Fixed to only use CLOCK_MONOTONTIC for FreeBSD and OpenBSD.